03-objects-and-composition/exercise/06.storeCatalogue.js

```diff
@@ -0,0 +1,16 @@
+function solve(arr) {
+
```

Comment:
```suggestion
const catalogue = new Map();
```

Reply:
Create a new Map() object that only collects unique products with their last updated price.